model/src/main/kotlin/config/RuleViolationResolutionReason.kt

```diff
@@ -25,6 +25,11 @@ enum class RuleViolationResolutionReason {
      */
     DYNAMIC_LINKAGE_EXCEPTION,
 
+    /**
+     * The rule violation is due to an inclusion of example code into a document and is acceptable in this case.
```

Comment:
Nit: Can we come up with something different than "document" here? Maybe just "file"?